core/src/main/java/io/aiven/klaw/dao/migration/MigrateData2x2x0.java

```diff
@@ -32,27 +35,40 @@ public MigrateData2x2x0(SelectDataJdbc selectDataJdbc, UpdateDataJdbc updateData
   }
 
   @MigrationRunner()
-  public boolean migrate() {
+  public boolean migrate() throws KlawDataMigrationException {
     List<UserInfo> allUserInfo = selectDataJdbc.selectAllUsersAllTenants();
     Set<Integer> tenantIds =
         allUserInfo.stream().map(UserInfo::getTenantId).collect(Collectors.toSet());
 
     for (int tenantId : tenantIds) {
       List<Team> teams = selectDataJdbc.selectAllTeams(tenantId);
+      List<UserInfo> users = selectDataJdbc.selectAllUsersInfo(tenantId);
+      Optional<UserInfo> superAdmin =
+          users.stream()
+              .filter(
+                  user ->
+                      user.getRole().equalsIgnoreCase(RolesType.SUPERADMIN.name())
```

Comment:
there might be a different naming for the role. It would be better if we look for that particular permission, and then derive role from it.